.bldr.toml

```diff
@@ -136,3 +139,15 @@ paths = [
   "components/core/*",
   "components/builder-depot-client/*",
 ]
+
+[hab-pkg-aci]
+paths = ["components/pkg-aci/*"]
+
+[hab-pkg-export-docker]
+paths = ["components/pkg-export-docker/*"]
```

Comment:
You can also add:

* components/core/*
* components/common/*
* components/hab/* (soon to be dropped, I need to move a module over first)

Reply:
Will do!